extensions/amp-video/0.1/flexible-bitrate.js

```diff
@@ -344,3 +351,15 @@ function getBufferedPercentage(videoEl) {
   }
   return bufferedSum / videoEl.duration;
 }
+
+/**
+ * Whether the video is loaded (so it should not be downgraded)
```

Comment:
Re: describe what the function does, as how/where it's used can change and make this comment obsolete.